api/nakadi-event-bus-api.yaml

```diff
@@ -1647,6 +1695,18 @@ definitions:
         type: string
         description: |
           Offset of the event being pointed to.
+      shift:
```

Comment:
Do you think it makes sense to include it in original cursor definition? I'm just afraid that 95% of users who don't need this functionality at all will be confused with this "shift".
Or do you plan that it will be possible to pass such a cursor directly to streaming endpoint? If yes - then I don't get why we need a separate endpoint that will shift cursors.